### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by VineethReddyAllam**

### CODE REVIEW

File: F21/vineethreddyallam/Code Review.txt [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+import java.util.Locale;
+
+public class Main
+{
+	public static void main(String[] args) {
+	    //English language
+	            String strUpper, strLower;
+        Scanner scan = new Scanner(System.in);
+
```
Comment (manojv17) [inline F21/vineethreddyallam/Code Review.txt:14]: Can use a choice or switch case to do perform each activity  in a simple way

File: F21/vineethreddyallam/Code Review.txt [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        
+        System.out.println("Enter one word/name in Uppercase : ");
+        toUpper = sca.nextLine();
+        
+        toLower = toUpper.toLowerCase();
+        System.out.println("Equivalent Word/Name in Lowercase : " + toLower);
+
+
+
+        //Irish Language
```
Comment (manojv17) [inline F21/vineethreddyallam/Code Review.txt:51]: Yes this looks better, more efficient way of using if else loops made easier

File: F21/vineethreddyallam/Code Review.txt [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+         Scanner sc = new Scanner(System.in);
+        System.out.println("enter the word:");
+         char[] sa = sc.next().toCharArray();
+            String str1 = new String(sa);
+        int n = str1.length();
+        char ch = str1.charAt(0);
+        char sech = str1.charAt(1);
+        if(ch == 't' || ch == 'n')
+        {
+         if(sech == 'A' || sech == 'E' || sech == 'I' || sech == 'O' || sech == 'U'){
```
Comment (manojv17) [inline F21/vineethreddyallam/Code Review.txt:61]: Can use a lower case vowels too

File: F21/vineethreddyallam/Code Review.txt (hunk 1 above)
Comment (VineethReddyAllam) [reply F21/vineethreddyallam/Code Review.txt:14]: Sure buddy, I'll make it

**Pull request closed without merging**